docs/index.md

```diff
@@ -15,46 +15,50 @@ In order to integrate with Lead Safe, the health provider will need to make a RE
-| address2       | Text     | Additional address information, e.g. apartment number      | patientprofile.address2                                                                                                                                                                                                     |
-| city           | Text     |                                                            | patientprofile.city                                                                                                                                                                                                         |
-| state          | Text     |                                                            | patientprofile.state                                                                                                                                                                                                        |
-| zip            | Text     | 5 digit zip code                                           | patientprofile.Zip                                                                                                                                                                                                          |
-| date_of_birth  | Date     | yyyy-mm-dd                                                 |                                                                                                                                                                                                                             |
-| gender         | Text     | M/F/U                                                      |                                                                                                                                                                                                                             |
-| race           | Text     | Code Values from Centricity                                | Standard ONC Race definitionsCode     description1002-5  American Indian or Alaska Native2028-9  Asian  2054-5  Black or African American 2076-8  Native Hawaiian or Other Pacific Islander 2106-3  White UNK       Unknown |
-| ethnicity      | Text     | Code Values from Centricity                                | Standard ONC Ethnicity definitionsCode     description2135-2  Hispanic or Latino  2186-5  Non Hispanic or Latino UNK      Unknown                                                                                           |
-| Visit Array    |          |                                                            |                                                                                                                                                                                                                             |
```

Comment:
I would clarify that this array may be empty. Should it be optional? In that case, missing means no visits. Ditto for the lab array.